base/permuteddimsarray.jl

```diff
@@ -140,7 +141,7 @@ julia> permutedims(X)
  [5 6; 7 8]  [13 14; 15 16]
 
 julia> transpose(X)
-2×2 Array{Array{Int64,2},2}:
+2×2 LinearAlgebra.Transpose{LinearAlgebra.Transpose{Int64,Array{Int64,2}},Array{Array{Int64,2},2}}:
```

Comment:
Will this doctest pass without a `using LinearAlgebra` prior to this call?

Reply:
Yes, at least for now, since `transpose` is `Base.transpose` and `LinearAlgebra` is `require`d in `sysimg.jl`. But I am not sure how informative this example actually is and would be happy to remove the showcasing of `transpose` from the `permutedims` examples. Would you, or anyone else, object to such a removal?

Reply:
I am more or less ambivalent :). Perhaps seek additional parties' opinions?

Reply:
Lets keep this for now then.